flang/tools/f18/f18.cpp

```diff
@@ -314,14 +312,6 @@ std::string CompileFortran(std::string path, Fortran::parser::Options options,
         nullptr /* action before each statement */, &asFortran);
     return {};
   }
-  if (driver.dumpPreFirTree) {
```

Comment:
Is there a replacement for this functionality?  That is, is it still possible to invoke
  $ bbc -fdebug-dump-pre-fir -emit-fir file.f90
and get PFT output as well as a file.mlir containing "high-level" FIR?

Reply:
Yes. I ported 3 of the 4 tests.